lib/src/dom/streams.dart

```diff
@@ -247,10 +764,41 @@ extension type Transformer._(JSObject _) implements JSObject {
   external set writableType(JSAny? value);
   external JSAny? get writableType;
 }
+
+/// The **`TransformStreamDefaultController`** interface of the
+/// [Streams API](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API)
+/// provides methods to manipulate the associated [ReadableStream] and
+/// [WritableStream].
+///
+/// When constructing a [TransformStream], the
+/// `TransformStreamDefaultController` is created. It therefore has no
+/// constructor. The way to get an instance of
+/// `TransformStreamDefaultController` is via the callback methods of
+/// [TransformStream.TransformStream].
 extension type TransformStreamDefaultController._(JSObject _)
     implements JSObject {
+  /// `Streams API`
```

Comment:
Again might not be anything we can do, but is this something we can remove?

Reply:
I was able to find the issue here (in the mdn scrapping code), and added a test for the fix.